pkg/util/net/net.go

```diff
@@ -72,3 +89,165 @@ func setBuffers(rc syscall.RawConn, sz int) error {
 
 	return err
 }
+
+type PrivateZoneDeletionConfig struct {
```

Comment:
Do we make any distinction between Delete and Remove in this context? Should this be renamed to PrivateZoneRemovalConfig or something? 

The reason I ask is because we've a mix of remove/delete in this file and wanted to ensure I didn't miss something

Reply:
Right. Changed now. Thanks!